src/collectors/aerospike/aerospike.py

```diff
@@ -61,8 +63,15 @@ def get_default_config(self):
             'objects',
             'client_connections',
             'index-used-bytes-memory',
+            # 3.9 Stats
+            'objects',
```

Comment:
Yeah was aware of that.  Though it's slightly less efficient, I thought it was clearer/easier to show and track exactly what was deemed useful out of the box in each version since they're obviously subject to change again in the next dialect change.  In fact, I'm nearly positive they'll get rid of the -'s in the near future.  If you have a scenario where something is in v2.7 and 3.9 but not v3.14, you're back to duplicating or making it more complicated.

I could throw them in a set to offset the cost of the 'in' tests and keep the maintainability and clarity.  Thoughts?